kaldb/src/main/proto/schema.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package slack.proto.kaldb.schema;
+
+option java_package = "com.slack.kaldb.proto.schema";
+
+message KaldbSchema {
```

Comment:
Instead of a `KalDBSchema` a better name would be `DatasetSchema` and associate a schema with a dataset.

Reply:
yup. Will change